includes/checkout/template.php

```diff
@@ -1099,5 +1099,30 @@ function edd_receipt_show_download_files( $item_id, $receipt_args, $item = array
 		}
 	}
 
-	return apply_filters( 'edd_receipt_show_download_files', $ret, $item_id, $receipt_args, $item );
+	if ( has_filter( 'edd_receipt_show_download_files' ) ) {
+		$cart = array();
```

Comment:
This isn't needed, as it's overwritten in the `if` block below, and isn't used outside of there.